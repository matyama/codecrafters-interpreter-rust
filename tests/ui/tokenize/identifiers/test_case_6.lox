_1236ar bar world_ baz _hello
