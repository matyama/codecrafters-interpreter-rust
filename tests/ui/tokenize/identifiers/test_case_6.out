IDENTIFIER _1236ar null
IDENTIFIER bar null
IDENTIFIER world_ null
IDENTIFIER baz null
IDENTIFIER _hello null
EOF  null
